packages/ia-zendesk-help-widget/src/styles/zendesk-widget.js

```diff
@@ -0,0 +1,62 @@
+import { css } from 'lit-element';
+
+export default css`
+  :host {
+    --buttonBlue: #194880;
+    --white: #fff;
+
+    --iconFillColor: var(--white);
+    --linkColor: var(--white);
+  }
+
+  .help-widget {
+    width: 108px;
+    height: 46px;
+    margin: 14px 20px;
+    position: fixed;
```

Comment:
Since help button rendering on DOM independently at right-bottom corner, not sure how without fixed position we can make this happen?

Reply:
You can let the consumer set the positioning. You can still use `fixed`, but it should be the consumer (petabox) that sets that, not the component itself.

Reply:
Since this component is for render overlayed help button and most of the help icon renders at bottom-right corner so it doesn't make sense to use another property for just positioning. so I am keeping this as it is.